amundsen_application/static/package.json

```diff
@@ -514,7 +514,7 @@
     },
     "extends": "stylelint-config-standard"
   },
-  "husky": {
+  "__husky": {
```

Comment:
Needs a fix added on https://github.com/lyft/amundsenfrontendlibrary/pull/480 to work properly

Reply:
What's the context?

Reply:
Some conflicts between ESLint and Prettier about whitespace on the TableIssues file.